### PR TITLE
Feature/mds 2861 frontend should gracefully handle 'inactive' status codes

### PR DESCRIPTION
**Pull request by mhuseinov**

### CODE REVIEW

File: services/core-api/app/api/compliance/models/compliance_article.py [hunk 1]
```diff
@@ -29,5 +29,5 @@ def find_by_compliance_article_id(cls, id):
         return cls.query.filter_by(compliance_article_id=id).first()
 
     @classmethod
-    def get_active(cls):
+    def get_all(cls):
```
Comment (nayr974) [inline services/core-api/app/api/compliance/models/compliance_article.py:32]: Thanks for renaming this!

File: services/core-api/app/api/mines/reports/models/mine_report_definition.py [hunk 2]
```diff
@@ -60,9 +60,9 @@ def find_by_mine_report_definition_guid(cls, _id):
     @classmethod
     # TODO: We need to revisit how we return "static content" items via a "get active only" function. This means
     # that some content in the UI that depends on now "inactive" content will fail to render and can cause crashes.
-    def get_active(cls):
+    # the above comment may be not relevant anymore :)
```
Comment (nayr974) [inline services/core-api/app/api/mines/reports/models/mine_report_definition.py:63]: Can you please remove the TODO and your comment?

Comment (mhuseinov) [reply services/core-api/app/api/mines/reports/models/mine_report_definition.py:63]: sure

File: services/core-web/common/utils/helpers.js [hunk 3]
```diff
@@ -38,8 +38,13 @@ export const createItemMap = (array, idField) => {
 // Function create id array for redux state. (used in src/reducers/<customReducer>)
 export const createItemIdsArray = (array, idField) => array.map((item) => item[idField]);
 
-export const createDropDownList = (array, labelField, valueField) =>
-  array.map((item) => ({ value: item[valueField], label: item[labelField] }));
+export const createDropDownList = (array, labelField, valueField, isActiveField = false) => {
```
Comment (nayr974) [inline services/core-web/common/utils/helpers.js:41]: 👍

**Pull request merged**